internal/cli/terraform.go

```diff
@@ -68,7 +68,7 @@ func (i *terraformInputs) parseResourceFetchers(api *auth0.API) ([]resourceDataF
 			fetchers = append(fetchers, &clientResourceFetcher{api})
 		case "auth0_client_grant":
 			fetchers = append(fetchers, &clientGrantResourceFetcher{api})
-		case "auth0_connection":
+		case "auth0_connection", "auth0_connection_clients":
```

Comment:
The `auth0_connection_clients` resource, along with other relationship resources, is outputted along with the `auth0_connection` resource. The user does not have the option to specify one or the other, simply all or  nothing. However, it may be confusing for users which want to specify these resources in the `--resources` argument. This change simply prevents any error that could occur.